activitysim/abm/models/util/test/test_school_escorting_utils.py

```diff
@@ -0,0 +1,76 @@
+# ActivitySim
+# See full license in LICENSE.txt.
+import os
+from ast import literal_eval
+import pandas as pd
+import numpy as np
+import pandas.testing as pdt
+
+from activitysim.abm.models.util.school_escort_tours_trips import (
+    create_bundle_attributes,
+    create_child_escorting_stops,
+    create_chauf_trip_table,
+)
+
+
+def test_create_bundle_attributes():
+    data_dir = os.path.join(os.path.dirname(__file__), "data")
+
+    inbound_input = pd.read_pickle(
+        os.path.join(data_dir, "create_bundle_attributes_inbound__input.pkl")
+    )
+    inbound_expected = pd.read_pickle(
+        os.path.join(data_dir, "create_bundle_attributes_inbound__output.pkl")
```

Comment:
Can we use parquet instead of pickle for test data?  Pickle is proven to have compatibility issues when changing version of Python or dependency (like pandas).

Reply:
As discussed during the meeting today, the parquet format does not preserve the dtype of the column like Pickle does. This is relevant in particular for the school escorting model because some columns contain arrays of values (like escorting participants or the school destinations of the escort bundle for example).  These arrays are not stored appropriately by the parquet format. 

The long-term solution agreed upon was to push this issue of parquet compatability (which I think happens in a couple of other places besides the school escort model) to the data model.

In order to make sure that this isn't an issue that was going to pop up immediately, the request was to test with pandas 2.  I can confirm that this unit test passes with the latest version of Pandas v2.2.2.